src/test/java/org/openrewrite/javascript/tree/UnaryTest.java

```diff
@@ -72,5 +73,17 @@ void keyofKeyword() {
         );
     }
 
-
+    @Issue("https://github.com/openrewrite/rewrite-javascript/issues/82")
+    @Test
+    void spreadOperator() {
```

Comment:
Info: A spread operator occurs in a tuple. There is a test for a spread operator for a Tuple in TupleTest that may be used